aws/resource_aws_efs_file_system_test.go

```diff
@@ -667,10 +667,10 @@ resource "aws_efs_file_system" "test" {
     AcceptanceTest = "PagedTags"
     CreationToken  = "radek"
     PerfMode       = "max"
-    Region         = "us-west-2"
+    Region         = %[2]q
```

Comment:
Since the tag key/value here seems arbitrary, let's just change them out or remove them instead of introducing the additional complexity of passing through the environment variable into the configuration.

Reply:
Sounds good.